akka-remote/src/test/scala/akka/remote/artery/tcp/ssl/TlsResourcesSpec.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.remote.artery.tcp.ssl
+
+import java.security.cert.CertificateFactory
+import java.security.cert.X509Certificate
+
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+import scala.util.Try
+import scala.util.control.NonFatal
+
+import akka.util.ccompat.JavaConverters._
+
+/**
+ *
+ */
+class TlsResourcesSpec extends AnyWordSpec with Matchers {
```

Comment:
This test ensures the files under `src/test/resources/ssl` follow certain properties (e.g. key algorythm, SAN values, key usage,...)